history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -1172,13 +1281,17 @@ he may receive a better crown among the blessed.
 
 Which is the prayer of, etc.
 
+--
+
 This address was received favorably, and therefore those of the yearly-meeting,
 which some time after was held at London,
 also drew up an address and some deputies of that meeting went to Windsor,
 where the court then was, and where W. Penn,
 one of those that had been chosen to present the address,
 made the following speech to the king;
 
+[quote]
```

Comment:
Not sure if this should be a quote because he gave this speech before he read the address.

Reply:
yeah, not sure either, but i think this is good.